.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release Pipeline
+
+on:
+  release:
+    types: [ published ]
```

Comment:
Do we also publish to NPM for prereleases? If so I think we need to add a `prerelease` type (not sure if it's the exact name).

Reply:
Yeah.

Thanks for the reminder!